submissions/Roophee/Popup_checkbox/css/main.css

```diff
@@ -0,0 +1,821 @@
+}
+
+.popup__item {
+  -webkit-box-sizing: content-box;
+          box-sizing: content-box;
+  display: -ms-grid;
+  display: grid;
+  justify-items: center;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  font-size: 0.9rem;
+  padding: 2px;
+  width: 60px;
+  height: 60px;
+}
+
+.popup__item-icon {
+  width: 30px;
+  height: 30px;
+  fill: #0c4e67;
+}
+
+.popup__item-icon:hover {
+  width: 35px;
+  height: 35px;
+  fill: #01161e;
+}
+
+.popup__item-icon:active {
+  width: 27px;
+  height: 27px;
+  fill: #01161e;
+}
+
+.popup__item:focus-within:not(.popup__item-last):not(.popup__item-last-link) {
+  border: 1px solid #033142;
+  -webkit-box-shadow: inset 0 0 2px #021d27;
+          box-shadow: inset 0 0 2px #021d27;
+}
+
+.popup__item-more-button {
+  -webkit-box-sizing: content-box;
+          box-sizing: content-box;
+  display: -ms-grid;
+  display: grid;
+  justify-items: center;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  font-size: 0.9rem;
+  padding: 2px;
+  width: 60px;
+  height: 60px;
+  position: relative;
+  -ms-grid-column: 2;
+  -ms-grid-column-span: 1;
+  grid-column: 2 / 3;
+  -ms-grid-row: 5;
+  grid-row: 5;
+  content: "some content";
+}
+
+.popup__item-more-button:focus-within {
+  border: 1px solid #033142;
+  -webkit-box-shadow: inset 0 0 2px #021d27;
+          box-shadow: inset 0 0 2px #021d27;
+}
+
+.popup__item-icon--more-button {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 12px;
+  top: 15px;
+}
+
+.popup__item-last {
+  -ms-grid-column: 1;
+  -ms-grid-column-span: 3;
+  grid-column: 1 / 4;
+  -ms-grid-row: 3;
+  grid-row: 3;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.popup__item-last-link {
+  padding: 7px 1px;
+  text-decoration: none;
+  color: #0c4e67;
+  font-size: 1.08rem;
+}
+
+.popup__item-last-link:hover {
+  color: #01161e;
+  font-size: 1.18rem;
+}
+
+.popup__item-last-link:active {
+  font-size: 0.9rem;
+}
+
+.popup__item-last-link:focus-within {
+  border: 1px solid #033142;
+  -webkit-box-shadow: inset 0 0 2px #021d27;
+          box-shadow: inset 0 0 2px #021d27;
+}
+
+.footer {
+  -ms-grid-row: 3;
+  -ms-grid-column: 1;
+  grid-area: footer;
+  border-top: 1px solid #033142;
+  -webkit-box-shadow: inset 0 0 2px #021d27;
+          box-shadow: inset 0 0 2px #021d27;
+  background-color: #227493;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+}
+
+.content-container {
+  background-color: #227493;
+  -ms-grid-row: 2;
+  -ms-grid-column: 1;
+  grid-area: content;
+  overflow: auto;
+  height: 800px;
+}
+/*# sourceMappingURL=main.css.map */
```

Comment:
could you please add an empty lane at the end of the file?

Reply:
I think, I did it. Please, make review again. Thanks!

Reply:
I mean , it was done after your review. Thanks).

Reply:
Fixed problems, added new lines, rename files. Make new review, please. Thanks.